cpu/em_smt_folding_test.py

```diff
@@ -31,6 +31,8 @@ class SmtFolding(Test):
     Throughput test for SMT folding in presence of swizzle.
     TODO :  add logic to  Revert back  value smt and cpu  state
             as it was before test.
+
+    :avocado: tags=cpu,power,privileged
```

Comment:
Can you add other `em` tests which does not include tags?

Reply:
yes sure   i will address that in another PR  i need to do rebase of this patch to add that

Reply:
Better to move this commit also in the new PR #1137 as you have already worked on it.

Reply:
:( sure but require rebase for this PR